module/project/templates/helper.js

```
@@ -327,6 +328,32 @@ module.exports = class ProjectTemplatesHelper {
+				// Check if organization extension exists for the loggedin user
+				let orgExtension
+				orgExtension = await orgExtensionQueries.orgExtenDocuments({
+					tenantId,
+					orgId,
+				})
+				// Create default org-extension policy if not found
+				if (!orgExtension || orgExtension.length === 0) {
+					orgExtension = await orgExtensionQueries.create({
+						tenantId,
+						orgId,
+						...CONSTANTS.common.DEFAULT_ORG_EXTENSION_POLICIES,
```

ESLint

[error] 'CONSTANTS' is not defined.

(no-undef)